client-react/src/QuickPulseQuery/QuickPulseQueryLayer.ts

```diff
@@ -129,14 +128,13 @@ export class QuickPulseQueryLayer {
     authorizationHeader: string,
     querySessionInfo: boolean,
     instanceId: string,
-    liveLogsSessionId?: string,
-    functionsRuntimeVersion?: string
+    useNewFunctionLogsApi: boolean,
+    liveLogsSessionId?: string
   ) {
-    const useNewFunctionLogsApi = this.useNewFunctionLogsApi(functionsRuntimeVersion);
     this._queryServersInfo = querySessionInfo;
 
     //note(stpelleg): Need to update the configuration each time we recieve a new session id
-    if (useNewFunctionLogsApi && liveLogsSessionId && this._detailedSessionInfo?.liveLogsSessionId !== liveLogsSessionId) {
+    if (!!useNewFunctionLogsApi && liveLogsSessionId && this._detailedSessionInfo?.liveLogsSessionId !== liveLogsSessionId) {
```

Comment:
Double negation is not required. `useNewFunctionsLogApi` is `boolean`. Consider doing it for `liveLogsSessionId` which is `string | undefined`.
```suggestion
    if (useNewFunctionLogsApi && !!liveLogsSessionId && this._detailedSessionInfo?.liveLogsSessionId !== liveLogsSessionId) {
```